pkgs/development/python-modules/d2to1/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage
+, lib
+, fetchPypi
+, nose
+}:
+buildPythonPackage rec {
+  pname = "d2to1";
+  version = "0.2.11";
+
+  checkInputs = [ nose ];
```

Comment:
> Ran 0 tests in 0.002s

You may need to explicitly set `checkPhase = "nosetests"` or something like that.

Reply:
it seems that only the github sources contain tests (not the pypi release). however tests fail on 0.2.11, i will also bump the version in a separate commit.